helm-charts/codegen-openshift-rhoai/tgi/templates/servingruntime-codellama-template.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  annotations:
+    opendatahub.io/apiProtocol: REST
+    opendatahub.io/modelServingSupport: '["single"]'
+  labels:
+    opendatahub.io/dashboard: "true"
+  name: tgi-codellama-7b-hf-gaudi
+  namespace: redhat-ods-applications
+objects:
+- apiVersion: serving.kserve.io/v1alpha1
+  kind: ServingRuntime
+  labels:
+    opendatahub.io/dashboard: "true"
+  metadata:
+    annotations:
+      openshift.io/display-name: Text Generation Inference CodeLlama-7b-hf on Gaudi
+      opendatahub.io/recommended-accelerators: '["habana.ai/gaudi"]'
+    name: tgi-codellama-7b-hf-gaudi
+  spec:
+    containers:
+      - args:
+          - --model-id
+          - /mnt/models/--meta-llama--CodeLlama-7b-hf/snapshots/b462c3c99b077d341db691ec780a33156f3c1472
+          - --port=8080
+          - --json-output
+          - --max-input-length
+          - "1024"
+          - --max-total-tokens
+          - "2048"
+        env:
+          - name: NUMBA_CACHE_DIR
+            value: /tmp/hf_home
+          - name: HF_HOME
+            value: /tmp/hf_home
+          - name: HF_HUB_CACHE
+            value: /mnt/models
+          - name: HUGGING_FACE_HUB_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: HUGGING_FACE_HUB_TOKEN
+                name: hf-token
+        image: ghcr.io/huggingface/tgi-gaudi:2.0.4
```

Comment:
I wonder if we should make it as a variable from values.yaml file. It would be easier to change version in future